model/xgboost/README.md

```diff
@@ -0,0 +1,62 @@
+# DFFML model_name Models
```

Comment:
Let's model this after https://raw.githubusercontent.com/intel/dffml/master/model/pytorch/README.md